@here/harp-examples/src/markers_screen-anchor.ts

```diff
@@ -13,6 +13,25 @@ import THREE = require("three");
 import { apikey } from "../config";
 
 export namespace GeoToScreenExample {
+    document.body.innerHTML += `
+        <style>
+        .message {
+            position: absolute;
+            float: right;
```

Comment:
does it really need the `float: right`, being already absolutely positioned?

Reply:
Nope, you're absolutely right, I fixed it.